addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties

```diff
@@ -117,6 +129,9 @@ channel-type.netatmo.lastStatusStore.description = Zeigt den Zeitpunkt an, an de
 channel-type.netatmo.timeUtc.label = Zeitpunkt Messung
 channel-type.netatmo.timeUtc.description = Zeigt den Zeitpunkt an, an dem die letzte Datenmessung stattgefunden hat.
 
+channel-type.netatmo.lastMessage.label = Zeitpunkt Meldung
```

Comment:
Just a reminder for myself: I maybe want to change this label to "Letzte Meldung".